vizro-core/docs/pages/user-guides/custom-actions.md

```diff
@@ -134,7 +137,7 @@ The following example shows how to create a custom action that shows the clicked
         Vizro().build(dashboard).run()
         ```
 
-        1. Just as for a normal Python function, the names of the arguments `show_species` and `points_data` are arbitrary and do not need to match on to the names of `inputs` in any particular way.
+        1. Just as for a standard Python function, the names of the arguments `show_species` and `points_data` are arbitrary and do not need to match on to the names of `inputs` in any particular way.
```

Comment:
Not sure what a normal or standard Python function is 😄  I think we mean any Python function right?

```suggestion
        1. Just as for any Python function, the names of the arguments `show_species` and `points_data` are arbitrary and do not need to match on to the names of `inputs` in any particular way.
```